lib/ilp.js

```diff
@@ -96,10 +102,26 @@ class Ilp {
 
     this.log.info('Connection created successfully')
 
-    await this.sendMoney({ ilpAmount: ilpScaledAmount, connection })
+    let { success, remainingAmount, error } = await this.sendMoney({ ilpAmount: ilpScaledAmount, connection })
+
+    // If we have sent NO money and the ilp sender failed, throw an error so we're notified of the failure
+    if (error && remainingAmount === ilpScaledAmount) {
+      throw new Error('ERROR: Encountered error when sending money via ILP: ', error)
+    }
+
+    // Simply log error, still continue noting the remaining amount but just log error for debugging purposes
+    if (error) {
+      this.log.info('ERROR: Encountered error when sending money via ILP: ', error)
```

Comment:
probably this should be `this.log.error`, although I don't remember how lambda treats output on stderr (seem to remember something about lambda failures that were really just console.errors)